docs/ELRS_DEVICES.md

```diff
@@ -0,0 +1,167 @@
+# mLRS Documentation: ELRS Devices #
+
+([back to main page](../README.md))
+
+## Receivers ##
+
+As of April 2024, ELRS receivers running the ESP82xx chipset are experimentally supported by mLRS.  The following receiver modules are recommended because they offer at least 100 mW transmit power.
+
+Note: ELRS receivers using an ESP32 chipset are currently not (yet) supported.
```

Comment:
Could we say they are being actively developed?

Reply:
Makes sense to me.